src/core/io_queue.cc

```diff
@@ -38,6 +38,10 @@
 #include <fmt/format.h>
 #include <fmt/ostream.h>
 
+#if FMT_VERSION >= 90000
+template <> struct fmt::formatter<seastar::fair_queue_ticket> : fmt::ostream_formatter {};
+#endif
+
```

Comment:
Ah! Even io-queue code wants to print ticket values, so it does deserve being in header

Reply:
yeah. see my comment. but this makes sense also.